doc/source/data/dask-on-ray.rst

```diff
@@ -31,6 +31,8 @@ workload. Using the Dask-on-Ray scheduler, the entire Dask ecosystem can be exec
 
      * - Ray Version
        - Dask Version
+     * - ``1.12.0``
+       - ``2022.2.0``
```

Comment:
I think this belongs to a different PR?

Reply:
Yes, I was trying to cherrypick that PR (merged to master already) but it's messing up this PR. I took it down for now.